packages/lab/src/toast/ToastClose.css

```diff
@@ -0,0 +1,8 @@
+/* Styles applied to ToastClose button */
+.saltButton.saltToastClose {
+  margin-right: calc(var(--salt-spacing-50) * -1);
```

Comment:
Do we need to use a negative margin here? They tend to cause more issues than they solve

Reply:
Hopefully not. There are still some design discussions taking place and there will be changes. I'll fix in the next PR if you're happy with the API on this one

Reply:
Will those design conversations also affect whether ToastClose stays?

Reply:
I don't think so. Close button is still optional, don't think there's any intention to change it. There was no mention about it.